report.md

```diff
@@ -60,6 +60,14 @@ Each of the following requirements will be linked to new tests, since no tests r
 | R1.2 |              Person without caps              |                                                         If there is at least one person that doesn't have any caps, there should be 0 ways to assign the caps. |
 | R1.3 |           No unique cap assignment            |                                  Assume there are >0 people and at least one cap per person. If there is no unique assignment of caps, the output should be 0. |
 | R1.4 |      One or more unique cap assignments       |                       Assume there are >0 people and at least one cap per person. If there is at least one unique assignment of caps, the output should be >0. |
+| R1.5 |               Too many people                 |                                                                       If there are too many people (i.e. capSets) then a ValueError should be raised.
```

Comment:
Right now R1.1 says there are either too many people or too few people, and R1.5 says there are too many people, which would imply there's an overlap among the tests. In reality, R1.1 only checks the case when there are too few people, which means there's actually no overlap. Could you update the description and the title of R1.1 to only mention the case when there are too few people? If so, the comments to that test case also need to be updated.